sharding/data_modeling.rst

```diff
@@ -10,6 +10,31 @@ Typically, you should choose that column as the distribution column which is the
 .. note::
   Citus also allows joining on non-distribution key columns by dynamically repartitioning the tables for the query. Still, joins on non-distribution keys require shuffling data across the cluster and therefore aren’t as efficient as joins on distribution keys.
 
-The best choice of distribution column varies depending on the use case and queries. Two common scenarios are the multi-tenant B2B application and the realtime analytics dashboard. In the former all tables include a tenant id and are distributed by it. When SQL queries are restricted to accessing data about a single tenant then Citus can execute them within a single shard. Having all data colocated in a shard is efficient and supports all SQL features. However running queries on a single shard limits the ability to parallelize execution.
+Use Cases and Tradeoffs
+-----------------------
 
-The other common option, realtime analytics, distributes by another column (such as user id). The queries in this scenario typically request aggregates information from multiple shards. This permits query parallelism but restricts some of the SQL features available, due to the constraints of being a distributed system.
+The best choice of distribution column varies depending on the use case and queries. Two common scenarios are the multi-tenant B2B application and the realtime analytics dashboard. Both cases involve tradeoffs.
+
+Multi-Tenant
+~~~~~~~~~~~~
+
+* ✔ SQL Coverage
+* ✔ Ease of Migration
+* ✘ Less Parallelism
+* ✘ Less effective for large (or few) tenants
+
+In the multi-tenant use case all tables include a tenant id and are distributed by it. When SQL queries are restricted to accessing data about a single tenant then Citus can execute them within a single shard. Having all data colocated in a shard is efficient and supports all SQL features. However running queries on a single shard limits the ability to parallelize execution.
+
+Existing schemas and queries typically require little adjustment during migration to the multi-tenant architecture. Additionally multi-tenant apps with few (or very large) tenants will not see a significant performance improvement.
+
+Real-Time Analytics
+~~~~~~~~~~~~~~~~~~~
+
+* ✔ Parallelism
+* ✔ Linear Scalability
+* ✘ Reduced SQL Coverage
+* ✘ Upfront Data Modeling
+
+The other common option, realtime analytics, distributes by another column (such as user id). The queries in this scenario typically request aggregate information from multiple shards. This permits query parallelism but restricts some of the SQL features available, due to the constraints of being a distributed system.
```

Comment:
Let's expand this content as well. Let's call out a use-case (e.g. real-time dash-boards), which then lays context for why we need parallelism and performance. We can point to our technical solution as an example. We can also add our existing paragraphs here. (The ones about statistics being gathered, join-performance and note on data shuffling).